android-sdk/publish.gradle

```diff
@@ -1,6 +1,6 @@
 apply plugin: 'maven-publish'
 
-version '1.2.0-pre0'
```

Comment:
is this correct ?

Reply:
The version number that gets committed to the repo doesn't matter. Publishing is done from a local repository. I changed this to an invalid version number in case somebody forgets to change it before publishing. I want to avoid a situation where somebody publishes a new release and overwrites one that is being used. This helps prevent things like there being multiple different 1.2.0 clients "in the wild" or inadvertently changing somebody's build without knowledge or consent.